src/developer/core/recording-format.md

```diff
@@ -62,6 +62,14 @@ And their corresponding timestamp files should follow the pattern:
 
 - `world_timestamps.npy`, `world_001_timestamps.npy`, `world_002_timestamps.npy`, ...
 
+### Video Player Compatibility
+
+In order to open the video files with the majority of video players, you will have to
+**transcode the video stream encoding from mjpeg to h264** and **change the color format
+from yuvj422p to yuv420p**. You can use the following [ffmpeg](https://ffmpeg.org/)
```

Comment:
I would recommend adding title attr to the link to ffmpeg for accessibility purposes. 

```suggestion
from yuvj422p to yuv420p**. You can use the following [ffmpeg](https://ffmpeg.org/ "FFmpeg - software solution for recording, converting, and streaming audio and video")
```